js/input_validation.js

```diff
@@ -0,0 +1,16 @@
+/**
+ *   This is the Input Validation function.
+ *   It checks that only lowercase is being typed into
+ *   the email input of the contact form.
+ */
```

Comment:
- [ ] Please, delete the commented lines, this could help to have a clean code and easy to read. 🐶

Reply:
Sorry, and please excuse me that contradict you. Those comments have a function and are needed to help to understand the code. They are part of the best practices and their use should be encouraged, not the opposite. 
[https://www.cs.utah.edu/~germain/PPS/Topics/commenting.html#:~:text=Commenting%20involves%20placing%20Human%20Readable,that%20other%20people%20will%20use.](URL)
`Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. Remember, well-documented code is as important as correctly working code.`